book/src/token-requests.md

```diff
@@ -4,7 +4,8 @@
 
 | Status | Min K8s Version | Max K8s Version |
 | ------ | --------------- | --------------- |
-| Alpha  | 1.20            | -               |
+| Alpha  | 1.20            | 1.20            |
+| Beta   | 1.21            | -               |
```

Comment:
In the feature gates section below, can you mention that it's on by default in 1.21

Reply:
done